src/domain/create-entry.spec.ts

```diff
@@ -743,6 +771,7 @@ function mockRulesetFiles(
     extractedModuleVersion?: string;
     metadataHomepage?: string;
     metadataVersions?: string[];
+    metadataYankedVersions?: { [version: string]: string };
```

Comment:
is there a schema file yet? I just did this for silo so maybe I'll make a JSON schema and then we'll get typescript typings for free

Reply:
I'm not aware of one.